packages/core/src/objects/maintenance_and_transport_object.ts

```diff
@@ -7,8 +7,8 @@ export class MaintenanceAndTransportObject extends AbstractObject {
   }
 
   public getAllowedNaming() {
-    return { // todo, verify
-      maxLength: 30,
+    return {
+      maxLength: 31,
```

Comment:
31 is odd

Reply:
but true. see include TTYPLENG. it's a mess...

If SAP wanted to save a ton of money long term they would have a global policy that all object keys are char250. #simplify